meerkat_frontend/src/files/pages/disease.html

```diff
@@ -29,9 +29,9 @@
 				<div class="chartBox box">
 					<div class="chartBox__heading" id="box_heading">
 					</div>
-					<div id="diseaseInfo" class="chartBox__content diseaseInfo">
+					<div id="diseaseInfo" class="chartBox__content diseaseInfo" style="min-height: 0px !important;">
```

Comment:
In general it is a good idea to avoid hard coding styling. Can you get these style changes into the _technical.scss file?